templates/utils.ts

```diff
@@ -30,3 +31,33 @@ export function filesToWebContainerFs(
   }
   return tree
 }
+
```

Comment:
I think the code here (`filesToVirtualFsTree` function) is somewhat repetitive above, but I'm not sure whether it should be merged with the functionality above, or if the whole approach should be redesigned.